nautobot_golden_config/views.py

```diff
@@ -789,60 +795,66 @@ class GoldenConfigSettingUIViewSet(NautobotUIViewSet):
     def get_extra_context(self, request, instance=None):
         """A GoldenConfig helper function to warn if the Job is not enabled to run."""
         jobs = []
-        jobs.append(
-            [
-                Job.objects.get(module_name="nautobot_golden_config.jobs", job_class_name="BackupJob"),
-                request,
-                constant.ENABLE_BACKUP,
-            ]
-        )
-        jobs.append(
-            [
-                Job.objects.get(module_name="nautobot_golden_config.jobs", job_class_name="IntendedJob"),
-                request,
-                constant.ENABLE_INTENDED,
-            ]
-        )
-        jobs.append(
-            [
-                Job.objects.get(module_name="nautobot_golden_config.jobs", job_class_name="DeployConfigPlans"),
-                request,
-                constant.ENABLE_DEPLOY,
-            ]
-        )
-        jobs.append(
-            [
-                Job.objects.get(module_name="nautobot_golden_config.jobs", job_class_name="ComplianceJob"),
-                request,
-                constant.ENABLE_COMPLIANCE,
-            ]
-        )
-        jobs.append(
-            [
-                Job.objects.get(module_name="nautobot_golden_config.jobs", job_class_name="AllGoldenConfig"),
-                request,
+        if constant.ENABLE_BACKUP:
```

Comment:
@jdrew82 this is what I was trying to say, what if backup is NOT enabled but deploy is, this would skip over that use case. 

I think that there is a misunderstanding on how add_message works.